sdk/identity/Azure.Identity/README.md

```diff
@@ -160,15 +160,42 @@ These examples demonstrate authenticating `SecretClient` from the [Azure.Securit
 
 #### Authenticate with a user-assigned managed identity
 
+To authenticate with a user-assigned managed identity, you must specify the ID of the managed identity.
+
+The following example demonstrates authenticating with a user-assigned managed identity using the client ID:
+
 ```C# Snippet:AuthenticatingWithManagedIdentityCredentialUserAssigned
-var credential = new ManagedIdentityCredential(clientId: userAssignedClientId);
+string userAssignedClientId = "some client ID";
+var options = new ManagedIdentityCredentialOptions() { ManagedIdentityId = ManagedIdentityId.FromUserAssignedClientId(userAssignedClientId) };
```

Comment:
Nit: Break up long lines like this in code samples. By doing so, you prevent horizontal scrolling in the README.

Reply:
fixed

Reply:
fixed